DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -186,9 +186,21 @@ private async Task FetchConfigAsyncWithTask()
             {
                 try
                 {
+                    var lastModified = res.Headers?.FirstOrDefault(e => e.Name?.ToLower() == "last-modified");
+                    if (configLastModified != "")
+                    {
+                        var parsedHeader = Convert.ToDateTime(lastModified);
+                        var storedHeader = Convert.ToDateTime(configLastModified);
+                        // negative means that the stored header is before the returned parsed header
+                        if (DateTime.Compare(storedHeader, parsedHeader) >= 0)
```

Comment:
any chance of adding a unit test for this?

Reply:
sure - but that would break the consistency of how all the other SDKs are done this way

Reply:
we'd also be unit testing something in the standard library; which I don't see much point in doing